netci.groovy

```diff
@@ -0,0 +1,83 @@
+// Import the utility functionality.
+
+import jobs.generation.Utilities;
+
+def project = 'dotnet/corefx'
+
+// **************************
+// Define code coverage build
+// **************************
+
+// TODO: Move this functionality into dotnet-ci once verified correct:
+
+def static addArchival(def job, def filesToArchive, def filesToExclude = '',
+    def doNotFailIfNothingArchived = false, def archiveOnlyIfSuccessful = true) {
+    
+    job.with {
+        publishers {
+            archiveArtifacts {
+                pattern(filesToArchive)
+                exclude(filesToExclude)
+                onlyIfSuccessful(archiveOnlyIfSuccessful)
+                allowEmpty(doNotFailIfNothingArchived)
+            }
+        }
+    }
+}
+
+def static addHtmlPublisher(def job, def reportDir, def name, def reportHtml, def keepAllReports = true) {
+    job.with {
+        publishers {
+            publishHtml {
+                report(reportDir) {
+                    reportName(name)
+                    keepAll(keepAllReports)
+                    reportFiles(reportHtml)
+                }
+            }
+        }
+    }
+}
+
+def static addPeriodicTrigger(def job, def cronString) {
+    job.with {
+        triggers {
+            cron(cronString)
+        }
+    }
+}
+
+// Define build string
+def codeCoverageBuildString = '''call "C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\Common7\\Tools\\VsDevCmd.bat" && build.cmd /p:Coverage=true'''
```

Comment:
Do we need to add /t:rebuild to force a rebuild/retest, or is that implicit in how this is used?